addon/lib/document.js

```diff
@@ -121,6 +121,10 @@ export default Base.extend({
       return Array.isArray(arr) ? arr.map((obj) => obj[key]) : null;
     });
     documentJexl.addBinaryOp("intersects", 20, intersects);
+    documentJexl.addTransform("debug", (any) => {
```

Comment:
What we could do is allowing an argument for debug that will be used to "label" the console debug: `|debug("test")` would result in `test: yourvalue`. As default we could use your proposed "JEXL Debug"

Reply:
Nice idea!

Reply:
Done

Reply:
Nice idea! Will replicate the same in the backend as well, then we can un-draft this and get it in master :+1: